src/components/learner-credit-management/AssignmentStatusTableCell.jsx

```diff
@@ -71,6 +72,12 @@ const AssignmentStatusTableCell = ({ enterpriseId, row }) => {
   }
 
   // Display the appropriate status chip based on the learner state.
+  if (isRetired) {
```

Comment:
nit: this comment is not quite as relevant to `isRetired`, as `isRetired` does not relate to "learner state" per se. Maybe move this comment above the other existing `learnerState` conditionals, and have a separate comment specific to `isRetired` here, e.g. (or similar):

```jsx
// Always display "Incomplete assignment" status chip for retired budgets
```